_content/monographs/bodipakkhiyadipani_sayadaw-ledi.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Bodipakkhiyadipani: The Manual of the Factors Leading to Enlightenment"
+authors:
+  - "Ledi Sayadaw"
+translator: :"U Sein Nyo Tun"
+external_url: "https://dhamma-library.cdn.winmetta.org/14.%20Rear-Buddhist-Books-2/Bodhipakhiyadipani.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/1MxjQl5_btTYcFG8Qc5JPJmJtKCX7wRYi/view?usp=drive_link"
```

Comment:
I can move this PDF file to the archive and we can replace this with the pdf and epub links of the wheel version?